src/components/profile/profile-logged-in.tsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import { useUrlState } from "../router/store";
+import { useI18nStore } from "../../i18n/i18n-store";
+import { useAuthStore } from "../../auth/auth-store";
+import Overview from "./overview";
+import AdoptedTrees from "./adopted-trees";
+import ProfileDetails from "./profile-details";
+import SecondaryButton from "../buttons/secondary";
+import PasswordReset from "./password-reset";
+
+const ProfileLoggedIn: React.FC = () => {
+  const i18n = useI18nStore().i18n();
+  const { logout } = useAuthStore();
+  const { url } = useUrlState();
+  const authType = url.searchParams.get("mode");
+
+  if (authType === "reset-password") return <PasswordReset />;
+  else {
```

Comment:
we don't need an else if we have a return in the if, I personally like this format:

```js
if (condition) {
  return ...
}

return ...
```